spec/safer_rails_console/console_spec.rb

```diff
@@ -17,21 +17,11 @@
     end
   end
 
-  context ".sandbox_prompt_user_input" do
```

Comment:
This should be in an integration test now that it's moved into a monkey-patch, but perhaps the code should be put back into this gem's classes to keep the patches as minimal asp possible.